### PR TITLE
fix included file to define Stretchy::VERSION

### PR DESCRIPTION
**Pull request by ultrasaurus**
also alphabetized list of requires to make finding missing one easier (queries was included twice!)

### CODE REVIEW

**Comment by ultrasaurus** [comment]
I just noticed that Solano CI is passing... not sure why that would be when rake spec showed this failure

File: spec/stretchy_spec.rb [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 describe Stretchy do
   it 'has a version number' do
+    puts "Stretchy: #{Stretchy.inspect} #{Stretchy.class}"
```
Comment (agius) [inline spec/stretchy_spec.rb]: Please remove debug output.

**Comment by agius** [comment]
Besides one comment above, :+1: :smiley_cat: Thanks for catching the double require - that was silly of me.

Not sure why `rake spec` was failing on your machine - passes on mine and Solano. Which versions of ruby and bundler are you using? RVM vs rbenv vs chruby ?

**Comment by ultrasaurus** [comment]
closed in favor of: https://github.com/hired/stretchy/pull/27

**Pull request closed without merging**